### ISSUE TITLE
Dropdown menu for "Forums" navbar tab

### ISSUE THREAD

**Issue by TheJoshue** [NamelessMC/Nameless#1740]
**Example**:
We create 1 forum/category and when we head to see changes on our site, we see this:
![image](https://user-images.githubusercontent.com/56804505/86513175-6a42cd00-be08-11ea-950b-843e9985d0fb.png)

We create 2+ forums/categories and when we head to see changes on our site, we see this:
*on hovering*
![image](https://user-images.githubusercontent.com/56804505/86513215-a2e2a680-be08-11ea-8c55-17d2fdac0bfe.png)

**Comment by StormyIceLeopard** [comment]
OOO, I like it. I would not have thought of that.

**Comment by Derkades** [comment]
or maybe like this
![image](https://user-images.githubusercontent.com/15892014/86513280-28feed00-be09-11ea-8fe3-c8cbabbb8615.png)

**Comment by StormyIceLeopard** [comment]
If it is done like that, isn't that stealing?

**Comment by TheJoshue** [comment]
> or maybe like this
> ![image](https://user-images.githubusercontent.com/15892014/86513280-28feed00-be09-11ea-8fe3-c8cbabbb8615.png)

I thought it might contain some fixed stuff, like "Unread Content", "Watched Threads", "Latest Posts" and "New post" (after clicking "New Post" > it will navigate to certain posting page where you can choose subforum/category to post, of course where user has permission to post), **and then** goes one break like ------------------, and just below it, there will be all categories (maybe better to add an option in forums module to enable/disable "List this forum/category in "Forums" navbar tab?").

**Comment by Derkades** [comment]
> If it is done like that, isn't that stealing?

No, spigotmc.org does not have copyright on having a forums dropdown with "Search Forums, Watched Forums, etc."

**Comment by TeemoCell** [comment]
Can be closed i guess?
